CHANGELOG.md

```diff
@@ -9,6 +9,10 @@
 - fix issue with video that do not need to be re-encoded and were marked as missing
 - README.md: replace course in sample command by one under a Creative Commons license
 
+# 1.0.2
```

Comment:
Do not add a version number yet, we keep everything in the ## Unreleased section until we know where it will be released (could be a patch release if we decide to release a patch, or a minor, or a major, this is not known yet).

Reply:
Noted, I'll remove it.

Reply:
Thank you